gcp_variant_transforms/bq_to_vcf.py

```diff
@@ -48,6 +54,9 @@
 
 _BASE_QUERY_TEMPLATE = 'SELECT * FROM `{INPUT_TABLE}`;'
 _COMMAND_LINE_OPTIONS = [variant_transform_options.BigQueryToVcfOptions]
+# The maximum number of variants to include in a single VCF file.
+_NUMBER_OF_VARIANTS_PER_SHARD = 1000
```

Comment:
I think this should be (1) a flag with a (potentially larger) default value and (2) named more generally as # of variants is misleading (I realize you mention 'maximum', but it can still lead to confusion about how this value is used; IOW, ppl may think that we split based on the number of variants rather than regions). What is the name of this variable in other contexts? I think I've see the word `bucket` used previously.

Reply:
Thanks! I added a flag as number_of_bases_per_shard (same as in the Java version) with default value 10000.